bin/build_project.sh

```diff
@@ -37,6 +37,10 @@ REPO_DIR=$AOMP_REPOS/$AOMP_PROJECT_REPO_NAME
 patchrepo $REPO_DIR
 
 # End check-openmp prep
+if [ "$AOMP_PROC" == "ppc64le" ] || [ "$AOMP_PROC" == "aarch64" ] ; then
+   echo "WARNING: You are about to build AOMP for the *untested* target $AOMP_PROC. Press enter to continue."
+   read
```

Comment:
if someone runs the build and redirects all output to a log file, will this hang waiting for a keyboard response ?

Reply:
Yes, it will hang. I only tested it with "|& tee". I will use sleep instead.